source/_integrations/securitas_direct.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: Securitas Direct
+description: Instructions on how to setup Securitas Direct home security system on Home Assistant.
+ha_category:
+  - Hub
+  - Alarm
+ha_release: 2020.12
+ha_iot_class: Cloud Polling
+ha_domain: securitas_direct
+---
+
+The `securitas_direct` integration allows users to manage their Securitas Direct installation
+There is currently support for the following device types within Home Assistant:
+- **Alarm Control Panel**: Reports on the current alarm status and can be used to arm and disarm the system.
+
+## Configuration
+
+To integrate Verisure with Home Assistant, add the following section to your `configuration.yaml` file:
+
+Securitas Direct can be integrated by adding the following `securitas_direct` section to your `configuration.yaml` file:
+
+## Configuration
+
+1. From Home Assistant, navigate to ‘Configuration’ then ‘Integrations’.
+1. Click the plus icon and type/select ‘Securitas Direct’.
+1. Enter your credentials and installation configuration.
+1. Click the Save button.
+
+{% configuration %}
```

Comment:
This is a YAML configuration block.

If you want to provide such a list, please use a configuration_basic block (examples can be found in Lovelace cards documentation in this repository).